src/client/libcurve_file.h

```diff
@@ -97,7 +97,8 @@ class FileClient {
      * @return: 返回文件fd
      */
     virtual int Open4ReadOnly(const std::string& filename,
-                              const UserInfo_t& userinfo);
+                              const UserInfo_t& userinfo,
+                              bool disableStripe = false);
```

Comment:
python cal need modify？

Reply:
unnecessary, currently `Open4ReadOnly` only for chunkserver/client read clone source file.

Reply:
unnecessary, currently Open4ReadOnly only for chunkserver/client read clone source file.